### DISCUSSION TITLE
Difference between from and sender

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2608] [Help with using Postal]
Hello,

What is the difference between "from" and "sender"?  Also, is there a way to have "from name" and "reply_to name"?

**Comment by sinnbeck** [comment]
You can set the sender header to am email with a domain on the server and then use any from address. Outlook (and probably others) will then display "[sender] on behalf of [from] "

Example borrowed from Google images 
![image](https://github.com/postalserver/postal/assets/13980140/59f08aa8-f0c9-47c3-b413-779b0c2b73c5)

Not sure about reply to name. Will need to test setting that header

**Comment by willpower232** [comment]
I believe you can pass names as part of the email address for example 

```"John Doe" <john.doe@example.com>```

and that should apply to all forms of email address when you're constructing a message.
